src/main/java/homework7/Task3/Calculator.java

```diff
@@ -0,0 +1,17 @@
+package homework7.Task3;
+
+public class Calculator {
+
+    public static int calculate(RomanNumber num1, RomanNumber num2, String operator) {
+
+        int result = 0;
+        switch (operator) {
+            case "+" -> result = num1.getWeight() + num2.getWeight();
```

Comment:
please consider defining `num1.getWeight()` and `num2.getWeight()` before the switch. it will make code cleaner